www/artists/afyfe/gallery.html

```diff
@@ -24,24 +24,27 @@
             </div>
             <div id="gallery">
                 <p>
-                    <a href="mossyrocks.html"><img src="images/mossyrocks.jpg" alt="Mossy Rocks" class='b1'></a> <a href="natureresting.html"><img src="images/natureresting.jpg" alt="Nature Resting" class='b1'></a>
+                    <a href="mossyrocks.html"><img src="images/mossyrocks.jpg" alt="Mossy Rocks" class="b1"></a> <a href="natureresting.html"><img src="images/natureresting.jpg" alt="Nature Resting" class="b1"></a>
                     <br>
-                    Mossy Rocks | Nature Resting
+                    Mossy Rocks | Nature Resting (psychedelic ambiguity of nature and self.)
                 </p>
                 <p>
-                    <a href="atthebeach.html"><img src="images/atthebeach.jpg" alt="At the Beach" class='b1'></a> <a href="morning.html"><img src="images/morning.jpg" alt="Morning" class='b1'></a>
+                    <a href="atthebeach.html"><img src="images/atthebeach.jpg" alt="At the Beach" class="b1"></a> <a href="morning.html"><img src="images/morning.jpg" alt="Morning" class="b1"></a>
                     <br>
                     On the Beach | Morning
                 </p>
                 <p>
-                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class='b1'></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class='b1'></a>
+                    <a href="prarielily.html"><img src="images/prarielily.jpg" alt="Prarie Lily" class="b1"></a> <a href="reflectionnebula.html"><img src="images/reflectionnebula.jpg" alt="The Reflection Nebula" class="b1"></a>
                     <br>
                     Prarie Lily | The Reflection Nebula
                 </p>
                 <p>
-                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.jpg" width="500" height="375" alt="no alt text"></a>
+                    <a href="rfiveproject1.html"><img src="images/afyfesinkpainting.JPG" width="500" height="375" alt="no alt text"></a>
                     <br>
-                    Tribute to the Regina 5
+                    (Summarizes the tribute to fyfe's grandfathers generation of artists. Is she cleaning them up? Or wiping them down the drain?) Tribute to the Regina 5
+                </p>
+                <p>
+                    "People always say you suffer for your artwork. ...I didn't know I was going to suffer this much," she said laughing.
```

Comment:
Did you get this from the Regina Leader Post? It feels a bit awkward for a website, particularly in context. The lines before give no indication of who "she" is, and nothing else is written in this format at all. It should probably just be a quote - remove the bit at the end, and make sure to add an attribution to Amber Fyfe with her name.

Reply:
Yes i did, wayne wanted a fitting quote from the leaderpost.

Reply:
Given that these quotes are just above or in the footer, proper formatting would probably be to make the quote into a single block of text (so things like "said Fyfe" don't interrupt it), and put the name of the source (in this case, Amber Fyfe) after the quote with a marker of some sort between the quote and the name.
Example:
I was born on the planet Earth - Wayne Tunison